lib/ratfun.gd

```diff
@@ -1077,6 +1077,10 @@ DeclareOperation( "UnivariateRationalFunctionByCoefficients",
 ##  The second version takes an univariate rational function and specializes
 ##  the value of its indeterminate to <A>val</A>.
 ##  Again, an optional argument <A>one</A> may be given.
+##  <C>Value( <A>upol</A>, <A>val</A> )</C> can also be expressed as <C>upol( 
+##  <A>val</A> )</C>. Those using this notation should remember that 
+##  composition of mappings in &GAP; still follows the conventions appropriate 
```

Comment:
I don't think we need this last bit, as polynomials are not composed via * anyway

Reply:
Ah okay, I'll remove it